src/components/Tweet.js

```diff
@@ -1,34 +1,20 @@
-function Tweet() {
+import Actions from "./Actions";
+import Message from "./Message";
+import ProfileImage from "./ProfileImage";
+import Timestamp from "./Timestamp";
+import User from "./User";
+
+function Tweet(props) {
```

Comment:
try destructuring props! 
you only need user, timestamp and message. image you can still get by user.image ^^

Reply:
> good job, try out some more with destructuring blush

Thanks for the feedback. Nowadays I do destructuring! :D